### ISSUE TITLE
[BUG]: Anonymous cards are not really anonymous 

### ISSUE THREAD

**Issue by jpjcorreia** [xgeekshq/split#1003]
## Description

Cards created with anonymous flag enabled are not really anonymous. If you inspect the payload (from opening the board) you can track who created the card by the createdBy field.

### Steps to reproduce

1. Create a card with "Post anonymously" enabled.
2. Refresh the page and inspect the payload from the split-be.azurewebsites.net/boards/:boardId
3. Using the createdBy property you can track who created the card.

![CleanShot 2023-02-02 at 16 21 34@2x](https://user-images.githubusercontent.com/28530831/216381917-40844eac-0edd-4971-ad2f-fd2ba5237b06.png)

### Expected result

What should happen?
The anonymous card should not have the createdBy property filled in or should be properly redacted in the database.

### Actual result

What happened.
The anonymous card it's not really anonymous because it's tied to the createdBy property

**Comment by nunocaseiro** [comment]
I think you can't see the _id if the card isn't created by you

**Comment by jpjcorreia** [comment]
Even if we can't, I'll try to test this with my team also, the connection between the card and user is somehow stored on the database. 
In my opinion, since this is anonymous this shouldn't happen. Thanks for the quick feedback.

**Comment by nunocaseiro** [comment]
Yeah, I understand your point. We need a way to know that a card was created by some user so he has privileges to edit and delete it. However, I'm sure we can figure out a way to handle this in a better way. I think we can talk soon about this topic
